src/js/components/Offer/index.scss

```diff
@@ -0,0 +1,4 @@
+.price {
+  color: #000;
+  font-weight: 400;
```

Comment:
There is `.font-weight-normal` for that.
For the color, I'm surpisd that there is none for `black`. What the color that you are overwriting here? (color: https://getbootstrap.com/docs/4.0/utilities/colors/)

Reply:
The color for links